src/main/java/org/opensearch/ad/ProfileUtil.java

```diff
@@ -59,11 +59,7 @@ public static void confirmDetectorRealtimeInitStatus(
         Client client,
         ActionListener<SearchResponse> listener
     ) {
-        SearchRequest searchLatestResult = createRealtimeInittedEverRequest(
-            detector.getDetectorId(),
-            enabledTime,
-            detector.getResultIndex()
-        );
+        SearchRequest searchLatestResult = createRealtimeInittedEverRequest(detector.getId(), enabledTime, detector.getCustomResultIndex());
```

Comment:
Nit: the old function name is unchanged (which is fine). What is the full acronym? Is "InittedEver" borrowed from somewhere else?

Reply:
Which function name are you referring to? Yes, InittedEver is from old code.